### DISCUSSION TITLE
windows powershell 프로그램 말고 더 다운받아야 하는 것이 있나요?

### DISCUSSION THREAD

**Discussion by Dwarveney** [devrel-kr/HackaLearn#121] [1 Q&A - HackaLearn 운영] [answered]
정적 웹 앱 워크샵 따라가는 중인데, powershell을 새로 깔고 git clone을 똑같이 썼더니 계속 오류가 나서요
혹시 특정한 패키지를 설치해야 하나요?
![오류](https://user-images.githubusercontent.com/88477651/128679607-f8b1288a-e309-4991-8f26-4395fcb61ea9.JPG)

**Comment by justinyoo** [comment, marked as answer]
현재 에러는 git 프로그램을 설치하지 않았거나, 설치했어도 경로를 찾을 수 없어서 나오는 문제입니다. https://git-scm.com/ 에서 다운로드 받아 설치하세요.

**Reply by justinyoo** [reply]
만약에 git 프로그램이 설치를 했는데도 같은 에러가 발생한다면 파워셸 콘솔을 관리자 권한으로 실행시켜서 한 번 해 보세요.

**Reply by Dwarveney** [reply]
git 새로 설치하니까 바로 됐습니다!! 감사합니다!!!

**Reply by justinyoo** [reply]
Answered 체크박스 체크해 주시면 다른 분들도 도움이 될 겁니다. 🥰
